pkg/didcomm/transport/http/inbound.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package http
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/pkg/errors"
+)
+
+// MessageHandler is a function that handles the inbound request payload
+// the payload will be unpacked prior to calling this function.
+// this function will be called for all incoming requests
```

Comment:
@Baha-sk I think comments on lines 20 and 21 can be removed

Reply:
ok..